docs/swagger.yaml

```diff
@@ -1562,18 +1490,19 @@ components:
             for more information.
         timeStamp:
           type: string
-          description: Timestamp in ISO-8601 representing when the order was captured.
+          description: Timestamp in ISO-8601 representing when the order was cancelled.
```

Comment:
make description generic for all three events "when operation was processed on the order"?

Reply:
Ah! Thanks.